cores/esp8266/umm_malloc/umm_malloc_cfg.h

```diff
@@ -17,9 +17,10 @@
 #include <stdint.h>
 #include <stddef.h>
 #include <stdbool.h>
-#include <debug.h>
+
 #include <pgmspace.h>
-#include <esp8266_undocumented.h>
+#include "../debug.h"
+#include "../esp8266_undocumented.h"
```

Comment:
It is sort of hard to remember why I cared about debug.h 2 month ago...

Having a PIO project with debug.h file in the tree, including Arduino.h caused it to include local debug.h instead of the one in the core/esp8266. I suppose a correct solution would be to either:
- use a sub-folder for debug.h e.g. include \<esp8266/debug.h\>)
- prefix it like the other headers, esp8266_debug.h

I removed other references as those aren't really necessary. This one, however, very quickly breaks when OOM debug is enabled

Reply:
I wouldn't mind if you rename it.

Reply:
I'll prepare a separate PR for that, since it is not necessary here.